src/librustc/dep_graph/graph.rs

```diff
@@ -875,23 +869,21 @@ impl DepGraph {
         }
     }
 
-    pub fn mark_loaded_from_cache(&self, dep_node_index: DepNodeIndex, state: bool) {
-        debug!("mark_loaded_from_cache({:?}, {})",
-               self.data.as_ref().unwrap().current.borrow().data[dep_node_index].node,
-               state);
+    pub fn mark_loaded_from_cache(&self, dep_node: DepNode, state: bool) {
+        debug!("mark_loaded_from_cache({:?}, {})", dep_node, state);
 
         self.data
             .as_ref()
             .unwrap()
             .loaded_from_cache
             .borrow_mut()
-            .insert(dep_node_index, state);
+            .insert(dep_node, state);
     }
 
+    // FIXME: Remove this since it appears to be unused?
```

Comment:
@michaelwoerister I'm guessing you're the person to ask about this.

Reply:
At some point we had debug output for incremental compilation that was using this. If it is really unused now, you can remove it.